packages/frontend-app-api/src/wiring/discovery.ts

```diff
@@ -23,19 +26,27 @@ interface DiscoveryGlobal {
 /**
  * @public
  */
-export function getAvailablePlugins(): BackstagePlugin[] {
+export function getAvailableFeatures(): (
+  | BackstagePlugin
+  | ExtensionOverrides
+)[] {
   const discovered = (
     window as { '__@backstage/discovered__'?: DiscoveryGlobal }
   )['__@backstage/discovered__'];
 
   return (
-    discovered?.modules.map(m => m.default).filter(isBackstagePlugin) ?? []
+    discovered?.modules.map(m => m.default).filter(isBackstageFeature) ?? []
   );
 }
 
-function isBackstagePlugin(obj: unknown): obj is BackstagePlugin {
+function isBackstageFeature(
+  obj: unknown,
+): obj is BackstagePlugin | ExtensionOverrides {
   if (obj !== null && typeof obj === 'object' && '$$type' in obj) {
-    return obj.$$type === '@backstage/BackstagePlugin';
+    return (
+      obj.$$type === '@backstage/BackstagePlugin' ||
+      obj.$$type === '@backstage/ExtensionOverrides'
+    );
```

Comment:
Could plugins provide overrides, or just the integrator can create them?

Reply:
Overrides would be separate from plugins, typically only created for internal customizations but there might also be fore example alternatives to core components or other extensions that are shipped through an override. I don't think we should allow plugins to provide overrides

Reply:
That makes sense, thanks for sharing your thoughts 🙌🏻